pkgs/tools/misc/vector/default.nix

```
@@ -118,9 +118,9 @@ rustPlatform.buildRustPackage {
   };
 
   meta = with lib; {
-    description = "A high-performance logs, metrics, and events router";
-    homepage = "https://github.com/timberio/vector";
-    license = with licenses; [ asl20 ];
+    description = "A high-performance observability data pipeline.";
```

Nit: We commonly remove the trailing period as suggested in [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

```suggestion
    description = "A high-performance observability data pipeline";
```

---

Thanks a lot!